tests/nightly/test_large_array.py

```diff
@@ -128,6 +138,38 @@ def test_take():
     assert np.sum(res[-1].asnumpy() == 1) == res.shape[1]
 
 
+def test_split():
+    a = nd.arange(0, LARGE_X).reshape(LARGE_X, 1)
+    b = nd.broadcast_to(a, shape=(a.shape[0], SMALL_Y))
+    outs = nd.split(b, num_outputs=10, axis=0)
+    for i, out in enumerate(outs):
+        assert np.sum(out[0].asnumpy() == i * out.shape[0]) == b.shape[1]
+    cat = nd.concat(*outs, dim=0)
+    assert np.sum(cat[-1].asnumpy() == LARGE_X) == cat.shape[1]
+    stack = nd.stack(*outs)
+    assert np.sum(stack[-1,-1,:].asnumpy() == LARGE_X) == b.shape[1]
+
+
+def test_argmin():
+    a = nd.arange(0, LARGE_X).reshape(LARGE_X, 1)
```

Comment:
Why do we need to broadcast? Can we just create a super long array and perform argmin?